src/io/flutter/sdk/FlutterSdkUtil.java

```diff
@@ -218,4 +220,16 @@ public static void setFlutterSdkPath(@NotNull final Project project, @NotNull fi
     // Fire events for a Flutter SDK change, which updates the UI.
     FlutterSdkManager.getInstance(project).checkForFlutterSdkChange();
   }
+
+  // TODO(devoncarew): The Dart plugin supports specifying individual modules in the settings page.
+
+  /**
+   * Do a best-effort basis to enable Dart support for the given project.
+   */
+  public static void enableDartSdk(@NotNull final Project project) {
+    final Module[] modules = ModuleManager.getInstance(project).getModules();
+    if (modules.length == 1) {
```

Comment:
Will keep this as is. I think as part of future work we'll want to expose controls for individual modules.